webapp/auction/models/auction.py

```diff
@@ -13,6 +13,14 @@ class Meta:
         verbose_name = 'Auction'
         verbose_name_plural = 'Auctions'
 
+    DIRECT_DEAL_TYPE = 0
+    DELIVERY_DEAL_TYPE = 1
+
+    DEALING_TYPE = (
+        (DELIVERY_DEAL_TYPE, 'delivery'),
+        (DIRECT_DEAL_TYPE, 'direct'),
+    )
```

Comment:
```suggestion
        (DIRECT_DEAL_TYPE, 'direct'),
        (DELIVERY_DEAL_TYPE, 'delivery'),
    )
```
p1: 이거 순서 뒤집어주시면 좋을 것 같아요. 순간 DELIVERY가 0인줄